bundles/org.openhab.binding.haassohnpelletstove/src/main/java/org/openhab/binding/haassohnpelletstove/internal/HaasSohnpelletstoveHandler.java

```diff
@@ -108,82 +108,70 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
-    /**
-     * Calls the service to update the oven data
-     *
-     * @param postdata
-     */
-    private boolean updateOvenData(@Nullable String postdata) {
-        Helper message = new Helper();
-        if (serviceCommunication.updateOvenData(postdata, message, this.getThing().getUID().toString())) {
-            updateStatus(ThingStatus.ONLINE);
-            return true;
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR,
-                    message.getStatusDesription());
-            return false;
-        }
-    }
-
     @Override
     public void initialize() {
         logger.debug("Initializing haassohnpelletstove handler for thing {}", getThing().getUID());
         config = getConfigAs(HaasSohnpelletstoveConfiguration.class);
-        boolean validConfig = true;
-        String errors = "";
-        String statusDescr = null;
-        if (config.refreshRate < 0 && config.refreshRate > 999) {
-            errors += " Parameter 'refresh Rate' greater then 0 and less then 1000.";
-            statusDescr = "Parameter 'refresh Rate' greater then 0 and less then 1000.";
-            validConfig = false;
+        if (config.refreshRate < 0 || config.refreshRate > 1000) {
```

Comment:
What does a refreshrate of 0? I guess it will hog your system continuously refreshing. Are you sure this is valid?

Reply:
Correct. changed it accordingly

Reply:
Please also update the `thing-types.xml` to have `min="1"`

Reply:
ok